mkosi/initrd.py

```diff
@@ -233,6 +233,13 @@ def main() -> None:
         action="store_true",
         default=False,
     )
+    parser.add_argument(
+        "--profile",
+        choices=InitrdProfile.values(),
+        nargs="*",
```

Comment:
This becomes ambiguous if we ever decide to add positional arguments to mkosi-initrd. Let's make this a `append` action instead so that `--profile` has to be specified for each profile.